### PR TITLE
feat: bitmex connection working for real accounts

### PR DESCRIPTION
**Pull request by muradmalik777**

### CODE REVIEW

File: src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js [hunk 1]
```diff
@@ -216,23 +216,21 @@ const ExchangeAccountAdd = ({ demo }) => {
               rules={{ required: true }}
             />
           )}
-          {typeOptions.length > 1 && (
```
Comment (cwagner22) [inline src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js:219]: I think we can delete it indeed

File: src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js [hunk 2]
```diff
@@ -205,8 +207,6 @@ const ExchangeAccountAdd = ({ demo }) => {
                     id: "accounts.exchange",
                   })}
                   onChange={(/** @type {string} **/ v) => {
-                    setValue("exchangeType", typeOptions[0].val);
-                    setValue("testNet", false);
```
Comment (cwagner22) [inline src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js:209]: Any reason you deleted that? I think we need to set to spot account when switching to kucoin for example, which doesn't have futures. That'll be an issue if previous selected exchange was binance with futures enabled.

File: src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js [hunk 3]
```diff
@@ -55,7 +55,9 @@ const ExchangeAccountAdd = ({ demo }) => {
   const testNet = watch("testNet");
   // Show testnet only for binance demo futures
   const showTestnet =
-    demo && exchangeType === "futures" && exchangeName.toLowerCase() === "binance";
+    demo &&
+    exchangeType === "futures" &&
+    (exchangeName.toLowerCase() === "binance" || exchangeName.toLowerCase() === "bitmex");
```
Comment (cwagner22) [inline src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js:60]: `["binance", "bitmex"].includes(exchangeName.toLowerCase())` might be shorter but not a big deal

File: src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js (hunk 2 above)
Comment (muradmalik777) [reply src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js:209]: @cwagner22  this was only used for the cases when the exchange type selection was hidden, but now as that select input is always shown, these values are initialized automatically when the select input is rendered, we don't need them here, its sort of a duplication.

Comment (cwagner22) [reply src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js:209]: For kucoin it's not shown since it's spot only.

Comment (muradmalik777) [reply src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js:209]: @cwagner22  for kucoin and bitmex, now its shown, the above checkI deleted, does that "typeOptions.length > 1", so now they are always shown, event its only one type.

**Pull request merged**